doc/release/1.17.0-notes.rst

```diff
@@ -83,6 +83,17 @@ The functions ``np.load``, and ``np.lib.format.read_array`` take an
 `allow_pickle` keyword which now defaults to ``False`` in response to
 `CVE-2019-6446 <https://nvd.nist.gov/vuln/detail/CVE-2019-6446>`_.
 
+Potential changes to the random stream
+--------------------------------------
+Due to bugs in the application of log to random floating point numbers,
+the stream may change when sampling from ``np.random.beta``, ``np.random.binomial``,
+``np.random.laplace``, ``np.random.logistic``, ``np.random.logseries`` or
+``np.random.multinomial`` if a 0 is generated in the underlying MT19937 random stream.
+There is a 1 in :math:`10^{53}` chance of this occurring, and so the probability that
+the stream changes for any given seed is extremely small. If a 0 is encountered in the
+underlying generator, then incorrect value (either ``np.inf``  or ``np.nan``) which
+was produced in rejected and dropped from the stream.
+
```

Comment:
"was produced in rejected and dropped" is not clear. Did you mean "was produced, **then** rejected and dropped"?

An aside: do we describe the use of a rejection-based algorithm in the overview or documentation of the distributions?

Reply:
Thanks, I clarified.  The specific algo isn't often discussed and so I removed "rejected" from the note.

Reply:
"then **the** incorrect value ... is **now** dropped" reads a little better I think